java/mapreduce/pom.xml

```diff
@@ -66,6 +66,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
     </dependency>
```

Comment:
Could you move this to the next section, `<!-- test inter-project -->`, please?

Reply:
Sure, thank you @dongjoon-hyun